### PR TITLE
User can buy a subscription

### PR DESCRIPTION
**Pull request by sundqvistjohan**
(Couldn't sort out pushing to Thomas PR...)
[PT - User can buy a subscription](https://www.pivotaltracker.com/story/show/170674152)

Adds Stripe and StripeForm component
Render StripeForm for non-subscriber on button click in DisplayCurrentArticle or Profile components

### CODE REVIEW

**Comment by EevanR** [comment]
LGTM

File: .env [hunk 1]
```diff
@@ -1 +1,2 @@
-REACT_APP_BASEURL = "https://reactive-herald-api.herokuapp.com/api/v1"
\ No newline at end of file
+REACT_APP_BASEURL = "https://reactive-herald-api.herokuapp.com/api/v1"
```
Comment (oliverochman) [inline .env]: add both .env files to `.gitignore`, everyone needs to have this locally. Share it in the project channel.

**Review by oliverochman — CHANGES_REQUESTED** [review]

File: cypress/integration/userCanPurchaseSubscription.feature.js [hunk 2]
```diff
@@ -0,0 +1,100 @@
...
+      response: "fixture:login_user.json"
+    });
+    cy.route({
+      method: "GET",
+      url: "http://localhost:3000/api/v1/auth/**",
+      response: "fixture:login_user.json"
+    });
+    cy.visit('/')
+    cy.get('#main-article-div').should('not.contain', 'Subscribe!')
+    cy.get("#loginButton").click();
```
Comment (tochman) [inline cypress/integration/userCanPurchaseSubscription.feature.js]: `kebab-case`please

Comment (sundqvistjohan) [reply cypress/integration/userCanPurchaseSubscription.feature.js]: Thanks, fixed

File: .env (hunk 1 above)
Comment (sundqvistjohan) [reply .env]: True, fixed

**Pull request merged**